pkg/docker/docker.go

```diff
@@ -29,13 +29,14 @@ func Tag(registry, image, tag string) string {
 }
 
 func ParseDockerignore(dir string) ([]string, error) {
-	var empty []string
+	var defaultIgnore = []string{"k8s"}
 	filePath := filepath.Join(dir, ".dockerignore")
 	if _, err := os.Stat(filePath); os.IsNotExist(err) {
-		return empty, nil
+		return defaultIgnore, nil
 	}
+
 	if file, err := ioutil.ReadFile(filePath); err != nil {
-		return empty, err
+		return defaultIgnore, err
 	} else {
 		var result []string
 		scanner := bufio.NewScanner(bytes.NewReader(file))
```

Comment:
Maybe result should be based on defaultIgnore rather than empty slice?

Reply:
Otherwise it would fail 😊